docs/reference/modules/discovery/azure.asciidoc

```diff
@@ -1,5 +0,0 @@
-[[modules-discovery-azure-classic]]
```

Comment:
If this page is disappearing entirely, it might be a good idea to add it in the https://www.elastic.co/guide/en/elasticsearch/reference/master/redirects.html

Reply:
TIL, thanks. Done in 400b2e46d68.